standalone/src/PythiaEvtGen_Interface.h

```diff
@@ -48,11 +48,10 @@ class PythiaEvtGen_Interface {
   void set_verbose(bool t=true)  {verbose=t;};
 
   void set_regenerate(bool t=true) {regenerate=t;};
-  
 
   void UpdatePythiaEvent(Pythia8::Particle *part1, EvtParticle *Evtpart );
-
-     
+  
+  void set_rehadronize(int i) { n_rehadronize=i;};
```

Comment:
Looks like this isn't needed:
```suggestion
  

```

Reply:
Well now it's not used now but it is indeed needed in future. Making long story short: Clement noticed that in some cases the rehadronization enters an infinite loop. So  I added a counter that after n times it will not create a requeted hadron it simulates a next event. This function is just a switch if a user wants to change the default number of attempts to create a requested hadron.